Dockerfile.aarch64

```diff
@@ -1,7 +1,7 @@
 # Original credit: https://github.com/jpetazzo/dockvpn
 
 # Smallest base image
-FROM aarch64/alpine:3.5
+FROM arm64v8/alpine:3.18.3
```

Comment:
What version of alpine do you have with a tag latest using aarch64? 
I am just following docker guidelines saying that the organization aarch64 is depreciated https://hub.docker.com/r/aarch64/alpine

Reply:
Sorry, i was not clear, I meant `arm64v8/alpine:latest`.

Reply:
Ah sorry I didn't understand that!
The problem with not pinning the docker image version is that nothing
guarantee that in the release of the alpine realse will not introduce a
breaking change (e.g. renaming of environmental variable or dependency
problem with the extra packages installed in the docker image).

Feel free to commit on my PR to change the version of the base image by the
latest you tested but I would avoid latest.

Best

On Mon, 27 Nov 2023, 20:46 Barna, ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In Dockerfile.aarch64
> <https://github.com/kylemanna/docker-openvpn/pull/759#discussion_r1406724134>
> :
>
> > @@ -1,7 +1,7 @@
>  # Original credit: https://github.com/jpetazzo/dockvpn
>
>  # Smallest base image
> -FROM aarch64/alpine:3.5
> +FROM arm64v8/alpine:3.18.3
>
> Sorry, i was not clear, I meant arm64v8/alpine:latest.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kylemanna/docker-openvpn/pull/759#discussion_r1406724134>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACNPBJ7QD26INSZ2U4SI3MDYGT3ZPAVCNFSM6AAAAAA5HOWTMCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONJRGMZDMMZQHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>